### ISSUE TITLE
Create a TwitterPreview component

### ISSUE THREAD

**Issue by andizer** [Yoast/javascript#48]
The twitter preview wrapper component (`TwitterPreview`) contains the whole twitter preview.

<img width="526" alt="yoast___yoast____twitter" src="https://user-images.githubusercontent.com/325040/49152566-8f5d8880-f313-11e8-9941-b0c9000c23c3.png">

## Properties
This component will receive:
* title
* description
* image
* siteName
* cardType (summary, summary with large image) 

Example:
```jsx
<TwitterPreview 
    title='The title' 
    description='The description to show' 
    image='image.jpg' 
    siteName='domain.com' >
   <TwitterTextContainer
     ...
   />
   <TwitterImage
     ...
   />
</TwitterPreview>
```

### Styling
It should have the following CSS props:
```css
    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
    width: 506px;
    min-height: 130px;
    border-radius: 12px;
    border: 1px solid #E1E8ED;
    box-sizing: border-box;
    color: #292F33;
    background: #FFFFFF;
```

When the element is hovered or active it should have a background of `#f5f8fa` and the border will get the color `rgba(136,153,166,.5)`

## CardType
When card type is summary, a square image should be loaded and will be placed on the left side. The width of every component (except the image) will be around the `357.562px`. 
If the large image card has been used the width will be `476px`. The only exception is if the ratio of the image being 1:1, then the summary mode will be active.

Parent of https://github.com/Yoast/javascript/issues/51
Parent of https://github.com/Yoast/javascript/issues/50
Parent of https://github.com/Yoast/javascript/issues/157

## To do
- [ ] Check the existing component and its props
- [ ] Implement the component with the correct styling
- [ ] Make sure the styling depends on the card type
- [ ] Write tests

**Comment by johannadevos** [comment]
Please note that there still seems to be an error in the official Twitter documentation for `Summary Card`: The (incorrect) example [Summary Card](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary) looks exactly the same as the (correct) example [Summary with large image Card](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image).

This problem was also signaled on ([StackOverflow](https://stackoverflow.com/questions/47578130/twitter-card-summary-vs-summary-large-image)); in #50 a correct example of a `Summary Card` is shown.

**Comment by maartenleenders** [comment]
We did some testing and discovered that the following exception
> If the large image card has been used the width will be 476px. The only exception is if the ratio of the image being 1:1, then the summary mode will be active.
is no longer correct. Square large images will still be displayed as the large image summary card. The top and bottom of the image will be cropped.

As per https://yoast.slack.com/archives/CSYSU835E/p1580732675006900, we will now only show the preview that is set in the general settings.

**Comment by GuidoOffermans** [comment]
closed by #477
